.buildkite/pipeline.yml

```diff
@@ -442,13 +442,17 @@
     - docker tag rayproject/ray:nightly-py37-cpu ray-ci:kuberay-test
     # Load the image into the kind node.
     - kind load docker-image ray-ci:kuberay-test
+    - echo "--- Setting up KubeRay operator."
+    - python python/ray/tests/kuberay/setup/setup_kuberay.py
     - echo "--- Running the test."
     - bazel test --config=ci $(./ci/run/bazel_export_options)
       --test_tag_filters=kuberay_operator
       --test_env=RAY_IMAGE=docker.io/library/ray-ci:kuberay-test
       --test_env=PULL_POLICY=IfNotPresent
       --test_env=KUBECONFIG=/root/.kube/config
       python/ray/tests/...
+    - echo "--- Tearing down KubeRay operator."
+    - python python/ray/tests/kuberay/setup/teardown_kuberay.py
```

Comment:
this should probably go into the cleanup clause (first row in list) and made idempotent so it's cleaned up regardless

Reply:
Added --ignore-not-found to kubectl delete to make it idempotent.
Tacked it on to the end of the cleanup function.